pkg/scheduler/metrics/metrics.go

```diff
@@ -87,6 +87,30 @@ var (
 			Name:      "total_preemption_attempts",
 			Help:      "Total preemption attempts in the cluster till now",
 		})
+	AssumeBindVolumesLatency = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Subsystem: schedulerSubsystem,
+			Name:      "assume_bind_volumes_latency_microseconds",
+			Help:      "Assume and bind volumes latency",
+			Buckets:   prometheus.ExponentialBuckets(1000, 2, 15),
+		},
+	)
+	VolumeBindingPredicateLatency = prometheus.NewHistogram(
```

Comment:
@bsalamat, what do you think about having a metric on a specific predicate?  This predicate does have the potential to be slow because it has to iterative through all PVs.

Reply:
Having metrics for a specific predicate is not consistent with the rest of our code base, but it's fine for now while we are finding a better solution for dynamic resource binding.